pkg/config/client.go

```diff
@@ -0,0 +1,93 @@
+package config
+
```

Comment:
Lets have the methods in this file named similar to this : https://github.com/openservicemesh/osm/blob/main/pkg/ingress/client.go